lib/devise.rb

```diff
@@ -153,7 +153,11 @@ module Test
   mattr_accessor :pepper
   @@pepper = nil
 
-  # Used to enable sending notification to user when their password is changed
+  # Used to send notification to the original user email when their email is changed.
+  mattr_accessor :send_email_change_notification
```

Comment:
It seems fine to deprecate now but leave it around for a while, only remove on a bigger bump. I might do that as a separate PR though. I'll rename this one to `email_changed`. Thanks!